site/app/AppKernel.php

```diff
@@ -16,7 +16,6 @@ public function registerBundles()
             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new Knp\Bundle\MarkdownBundle\KnpMarkdownBundle(),
```

Comment:
We can't remove the markdown bundle since our static page content wrapper template uses the markdown filter.